toucan_connectors/common.py

```diff
@@ -255,4 +255,22 @@ def convert_to_qmark_paramstyle(query_string: str, params_values: dict) -> str:
     extracted_params = re.findall(RE_NAMED_PARAM, query_string)
     qparams = [m[2:-2] for m in extracted_params]
     ordered_values = [params_values.get(p) for p in qparams]
-    return re.sub(RE_NAMED_PARAM, '?', query_string), ordered_values
+
+    # Check if we need to replace a list
+    for i, o in enumerate(ordered_values):
+        if type(o) is list:
```

Comment:
```suggestion
        if isintance(o, list):
```
est + conventionnel et permet de gérer aussi le cas de types qui héritent de `list`

Reply:
Thx je change ça